modules/cluster-logging-collector-log-forward-loki.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forward-loki_{context}"]
+= Forwarding logs to Grafana Loki
+
+You can forward logs to an external Loki logging system in addition to, or instead of, the internal default {product-title} Elasticsearch instance.
+
+To configure log forwarding to Grafana Loki, create a `ClusterLogForwarder` custom resource (CR) with an output to Loki and a pipeline that uses the output. The output to Loki can use the HTTP (insecure) or HTTPS (secure HTTP) connection.
+
+.Prerequisites
+
+* You must have an Loki logging system that is configured to receive the logging data using the specified protocol or format.
```

Comment:
```suggestion
* You must have a Loki logging system running at the URL specified in the custom resource.
```
There isn't anything to configure on the Loki side regarding protocol or format, we use Loki's REST API to forward logs.

Reply:
Updated. Thanks.